resources/migrations/019_add_schema_column_to_table.yaml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+  - changeSet:
+      id: 19
+      author: camsaul
+      changes:
+        - addColumn:
+            tableName: metabase_table
+            columns:
+              - column:
+                  name: schema
+                  type: VARCHAR
```

Comment:
Lets put character limit on this varchar.  Having an unbound varchar doesn't play as nicely with databases other than postgres and realistically we don't need this to be unbound.  From the postgres docs it looks like the max size for schema is 63 chars.